Dockerfile

```
@@ -34,7 +34,12 @@ COPY --from=builder --chown=app /app/static ./static
 
 COPY --chown=app ./requirements.txt /app/requirements.txt
 RUN pip install -r requirements.txt
+# When the user's Accept-Language is set to `fy`, Django's LocaleMiddleware
+# doesn't load `fy-NL`. This is a workaround to force the Frysian and Swedish
+# localisations to load anyway when appropriate.
 COPY --chown=app . /app
+RUN cp -r /app/privaterelay/locales/fy-NL/ privaterelay/locales/fy/
```

Would a link be enough?

```suggestion
RUN ln -s /app/privaterelay/locales/fy-NL/ privaterelay/locales/fy/
```

---

Looks point, looks like it would be: d18d8224